opds_import.py

```diff
@@ -943,7 +944,14 @@ def _get(self, url, headers):
         Long timeout, raise error on anything but 2xx or 3xx.
         """
         headers = dict(headers or {})
-        headers['Accept'] = OPDSFeed.ACQUISITION_FEED_TYPE
+
+        types = dict(
+            opds_acquisition=OPDSFeed.ACQUISITION_FEED_TYPE,
+            atom="application/atom+xml",
+            xml="application/xml",
+            anything="*/*",
+        )
+        headers['Accept'] = "%(opds_acquisition)s, %(atom)s;q=0.9, %(xml)s;q=0.8, %(anything)s;q=0.1" % types
```

Comment:
Standard Ebooks incorrectly serves its OPDS feed as XHTML. At the same time, we can't just get rid of the Accept header because without it, unglue.it will serve HTML instead of OPDS.